engine/access/rpc/logging_interceptor.go

```diff
@@ -0,0 +1,25 @@
+package rpc
+
+import (
+	grpczerolog "github.com/grpc-ecosystem/go-grpc-middleware/providers/zerolog/v2"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/logging"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tags"
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+func customClientCodeToLevel(c codes.Code) logging.Level {
+	if c == codes.OK {
+		// log successful returns as Debug to avoid excessive logging in info mode
+		return logging.DEBUG
+	}
+	return logging.DefaultServerCodeToLevel(c)
+}
+
+// loggingInterceptor creates the logging interceptors
+func loggingInterceptor(log zerolog.Logger) []grpc.UnaryServerInterceptor {
```

Comment:
right now this does nothing fancy and could be folded in the `rpc/engine.go` file but we can add more customization here for the logging specially for the Cadance script logging in the `ExecuteScript` calls.